File: backend/SNS.Application/Identity/SecuritySessions/Queries/GetUserSessions/SessionSummaryDto.cs
namespace SNS.Application.Identity.SecuritySessions.Queries.GetUserSessions;

public sealed record SessionSummaryDto(
    Guid UserId,
    Guid Id,
    string DeviceName,
    DateTime LoginAt,
    DateTime LastSeenAt,
    DateTime? LogoutAt,
    string Counrty,
    string City,
    int DurationMinutes,
    bool IsRevoked,
    string? RevokedReason);

File: SNS.Web/src/app/identity/account-settings/security-sessions/session-management/contracts/session-summary.dto.ts
export interface SessionSummaryDto {
    userId: string;
    id: string;
    deviceName: string;
    loginAt: string;
    lastSeenAt: string;
    logoutAt?: string;
    counrty: string;
    city: string;
    durationMinutes: number;
    isRevoked: boolean;
    revokedReason?: string;
}
